nipype/interfaces/bids_utils.py

```diff
@@ -1,147 +0,0 @@
-# -*- coding: utf-8 -*-
```

Comment:
I am okay with this change - but removing this is an api change from `0.14` @satra WDYT